### DISCUSSION TITLE
JMusicBot Discord setup on Raspberrypi

### DISCUSSION THREAD

**Discussion by Zestyclose-Thanks-82** [jagrosh/MusicBot#1402] [Questions]
Hi guys,
I have a question. Can someone help me with setting the JMusicBot up on my Raspberry Pi. I already set it up on my Windows pc, but it can't run 24/7, so I was hoping someone could help me set this thing up on my Raspberry Pi.
Here is some info:
https://jmusicbot.com
https://github.com/jagrosh/MusicBot
https://www.youtube.com/watch?v=FyOuyCzeEPo&t=483s&pp=ygUJam11c2ljYm90
Please, if anyone knows how I could set this up, please HELP ME.
Thanks!

**Comment by AdnanPSD** [comment]
just move your "JMusicBot.jar" and "config.txt" files  to raspberry pi /home/*username*/ and than open your cmd,
cd /home/*username*/
java -jar JmusicBot.jar
you can run like that if you want open automatically check it out "https://jmusicbot.com/running-as-a-service/"
